styles/stylesheet.css

```diff
@@ -0,0 +1,437 @@
+   @media print {
+     body img {
+        display: none;
+     } 
+   }
+   @media screen and (min-width: 200px) {
+       .homeimage {
+          height: 400px;
+          object-fit: contain;
+       }
+   }
+   @media screen and (min-width: 200px) {
+       .blogimage2 {
+          height: 200px;
+          object-fit: contain;
+          padding-top: 30px;
+       }
+   }
+   @media screen and (min-width: 200px) {
+       img {
+         display: inline;
+          width: 100%;
+       }
+   }
+   @media screen and (max-width: 768px) {
+        .hidetext {
```

Comment:
You'll want to take a closer look here. Per the assignment, we're looking for the blog previews on the home page to disappear when on mobile. You're on the right track with using display. Feel free to reach out if you need some help on this!